src/extensions/markdown/CodeBlock/CodeBlockSpecs/index.ts

```diff
@@ -15,6 +15,14 @@ export type CodeBlockSpecsOptions = {
     nodeview?: ExtensionNodeSpec['view'];
 };
 
+const getLangOfNode = (node: Element) => {
+    return (
+        node.getAttribute(CodeBlockNodeAttr.Lang) ||
+        node.firstElementChild?.getAttribute('class')?.split(' ')?.[1] ||
```

Comment:
Let's add additional checks that `firstElementChild` is `<code/>` element, it has `hljs` classname, etc ?